stdsimd/mod.rs

```diff
@@ -240,101 +241,6 @@
 /// we'll be using SSE4.1 features to implement hex encoding.
 ///
 /// ```
-/// #![feature(cfg_target_feature, target_feature, stdsimd)]
```

Comment:
I have no idea about what happened here. Did rustfmt delete all of this?

Reply:
Yep, it look like `rustfmt` just deleted all of this. @nrc does this look familiar?